### PR TITLE
Allow networking to work properly by matching the kernel version to the RaspiOS version

### PR DESCRIPTION
**Pull request by nateww**
In addition, fix the script so that the qemu-image resize actually works properly.  Finally, avoid some startup warnings by changing the image to be in raw mode.

### CODE REVIEW

File: native-emulation/install.sh [hunk 1]
```diff
@@ -13,7 +13,7 @@ readonly PTB_FILE="${TMP_DIR}/${PTB}"
 # commit hash to use for the https://github.com/dhruvvyas90/qemu-rpi-kernel/ repo:
 readonly COMMIT_HASH='c522bff346bb7401ad0b979778c23be52089618f'
 
-readonly IMAGE_URL="https://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2020-02-14/${IMAGE}.zip"
+readonly IMAGE_URL="https://downloads.raspberrypi.org/raspios_lite_armhf/images/raspios_lite_armhf-2020-08-24/${IMAGE}.zip"
 readonly KERNEL_URL="https://github.com/dhruvvyas90/qemu-rpi-kernel/blob/${COMMIT_HASH}/native-emulation/5.4.51%20kernels/${KERNEL}?raw=true"
```
Comment (nateww) [inline native-emulation/install.sh:17]: The 2020-02-13 kernel is a V4 kernel, and the 2020-08-24 release matches the 5.4.51 kernel version (KERNEL_URL).

File: native-emulation/install.sh [hunk 2]
```diff
@@ -51,7 +51,7 @@ extract_images () {
     curl -sSL "$IMAGE_URL" -o "${IMAGE}.zip"
   [ -f "${IMAGE}.img" ] || \
     unzip "${IMAGE}.zip"
-  [ -f "${IMAGE}.img" ] || \
+  [ -f "${IMAGE}.img" ] && \
```
Comment (nateww) [inline native-emulation/install.sh:54]: We want to resize the image file if the file exists, not if it doesn't exist.

File: native-emulation/run.sh [hunk 3]
```diff
@@ -28,8 +28,8 @@ run_qemu () {
     -device usb-kbd \
     -device 'usb-net,netdev=net0' \
     -netdev 'user,id=net0,hostfwd=tcp::5022-:22' \
-    -sd "$IMAGE_FILE" \
-    -dtb "$PTB_FILE" \
+    -drive "file=$IMAGE_FILE,index=0,format=raw" \
```
Comment (nateww) [inline native-emulation/run.sh]: Avoid warnings that say to use raw mode for the downloaded image.

File: native-emulation/install.sh (hunk 2 above)
Comment (faf0) [reply native-emulation/install.sh:54]: thanks!

File: native-emulation/run.sh (hunk 3 above)
Comment (faf0) [reply native-emulation/run.sh]: Thank you!

```suggestion
    -drive "file=${IMAGE_FILE},index=0,format=raw" \
```

**Review by faf0 — APPROVED** [review]
Thank you! Changes look good to me.

**Comment by nateww** [comment]
Thank you!

**Pull request merged**